kci_test

```diff
@@ -92,6 +92,9 @@ class cmd_get_lab_info(Command):
             'url': lab.url,
             'devices': lab_api.devices,
         }
+        dir = os.path.dirname(args.lab_json)
+        if not os.path.exists(dir):
+            os.makedirs(dir)
```

Comment:
This fails if the JSON file path is the current directory, because then `dir` is an empty string:
```
+ ./kci_test get_lab_info --lab=lab-baylibre --lab-json=lab-info.json --user=kernel-ci --token=****
Traceback (most recent call last):
 ​File "./kci_test", line 209, in <module>
   ​status = args.func(test_configs, lab_configs, args)
 ​File "./kci_test", line 98, in __call__
   ​os.makedirs(dir)
 ​File "/usr/lib/python3.7/os.py", line 221, in makedirs
   ​mkdir(name, mode)
FileNotFoundError: [Errno 2] No such file or directory: ''
```

Reply:
Easy enough to check that `dir` is non-empty.

I know it's common practice for scripts to have to mkdir before calling, but I find that clumsy and error prone.  It's simple enough for the script to "do the right thing", so we should, IMO.

Reply:
Alrighty :)